src/hooks/productData/llm_analytics.tsx

```diff
@@ -1,494 +1,528 @@
-import { IconBell, IconClockRewind, IconDashboard, IconGraph, IconHandMoney, IconLightBulb, IconListTreeConnected, IconLlmAnalytics, IconPiggyBank, IconRewindPlay, IconShield, IconSparkles, IconTarget, IconTrends, IconUser, IconWarning, IconGlobe, IconListCheck } from '@posthog/icons'
-import { IconAnthropic, IconGemini, IconGrid, IconHelicone, IconKeywordsAI, IconLangChain, IconLangfuse, IconOpenAI, IconOpenRouter, IconTag, IconTraceloop, IconVercel } from 'components/OSIcons'
+import {
+    IconBell,
+    IconClockRewind,
+    IconDashboard,
+    IconGraph,
+    IconHandMoney,
+    IconLightBulb,
+    IconListTreeConnected,
+    IconLlmAnalytics,
+    IconPiggyBank,
+    IconRewindPlay,
+    IconShield,
+    IconSparkles,
+    IconTarget,
+    IconTrends,
+    IconUser,
+    IconWarning,
+    IconGlobe,
+    IconListCheck,
+} from '@posthog/icons'
+import {
+    IconAnthropic,
+    IconGemini,
+    IconGrid,
+    IconHelicone,
+    IconKeywordsAI,
+    IconLangChain,
+    IconLangfuse,
+    IconOpenAI,
+    IconOpenRouter,
+    IconTag,
+    IconTraceloop,
+    IconVercel,
+} from 'components/OSIcons'
 
 export const llmAnalytics = {
-  name: 'LLM Analytics',
-  Icon: IconLlmAnalytics,
-  description: 'Track costs, performance, and usage of your AI features',
-  handle: 'llm_analytics',
-  slug: 'llm-analytics',
-  color: 'purple',
-  colorSecondary: 'green-2',
-  category: 'analytics',
-  slider: {
-    marks: [100000, 1000000, 10000000, 100000000],
-    min: 100000,
-    max: 100000000,
-  },
-  volume: 100000,
-  seo: {
-    title: 'LLM analytics - PostHog',
-    description:
-      'Track every conversation, model performance, costs, and errors in your LLM applications. 10x cheaper than other LLM observability tools.',
-  },
-  overview: {
-    title: 'Monitor and debug your AI products',
-    description:
-      'Analyze traces, spans, per-user costs, latency, and usage of your AI features',
-    textColor: 'text-white',
-    layout: 'overlay',
-  },
-  screenshots: {
-    overview: {
-      src: 'https://res.cloudinary.com/dmukukwp6/image/upload/llm_overview_desktop_2399cc57d6.png',
-      srcMobile: 'https://res.cloudinary.com/dmukukwp6/image/upload/llm_overview_mobile_b9565d0690.png',
-      alt: 'LLM analytics dashboard',
-      classes: '',
-      // imgClasses: 'rounded-tl-md shadow-2xl',
-      classesMobile: '',
-      imgClassesMobile: '',
-    },
-    home: {
-      src: 'https://res.cloudinary.com/dmukukwp6/image/upload/screenshot_llm_analytics_light_a436da72f7.png',
-      srcDark:
-        'https://res.cloudinary.com/dmukukwp6/image/upload/screenshot_llm_analytics_dark_d8f32c249b.png',
-      alt: 'LLM Analytics screenshot',
-      classes: 'justify-end items-end pl-4 @lg:pl-6',
-      imgClasses: 'rounded-tl-lg shadow-2xl',
+    name: 'LLM Analytics',
+    Icon: IconLlmAnalytics,
+    description: 'Track costs, performance, and usage of your AI features',
+    handle: 'llm_analytics',
+    type: 'llm_analytics',
```

Comment:
I literally added this line only, rest of changes are from the linter that runs on commit, is this expected?